app/scripts/app.js

```diff
@@ -269,6 +269,10 @@ angular
         templateUrl: 'views/about.html',
         controller: 'AboutController'
       })
+      .when('/commandLine', {
```

Comment:
sorry missed this before, we dont camel-case the URLs anymore, it should be command-line